tool/tctl/common/db_command.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"os"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/defaults"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/gravitational/teleport/lib/services"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+)
+
+// DbCommand implements "tctl db" group of commands.
+type DbCommand struct {
+	config *service.Config
+
+	// format is the output format (text, json, or yaml).
+	format string
+
+	// dbList implements the "tctl db ls" subcommand.
+	dbList *kingpin.CmdClause
+}
+
+// Initialize allows DBCommand to plug itself into the CLI parser.
+func (c *DbCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	c.config = config
+
+	db := app.Command("db", "Operate on databases registered with the cluster.")
+	c.dbList = db.Command("ls", "List all databases registered with the cluster.")
+	c.dbList.Flag("format", "Output format, 'text', 'json', or 'yaml'").Default("text").StringVar(&c.format)
+}
+
+// TryRun attempts to run subcommands like "apps ls".
```

Comment:
What is this method for? And why is this referring to `apps ls`?

Reply:
Because of copy-paste obviously :) As for the method, this is how other tctl subcommands are implemented as well (e.g. tctl apps, etc.).